save-cli/src/commonMain/kotlin/org/cqfn/save/cli/Config.kt

```diff
@@ -0,0 +1,53 @@
+/**
+ * Utilities to work with SAVE config in CLI mode
+ */
+
+package org.cqfn.save.cli
+
+import org.cqfn.save.core.config.ReportType
+import org.cqfn.save.core.config.SaveConfig
+
+import okio.Path.Companion.toPath
+
+import kotlinx.cli.ArgParser
+import kotlinx.cli.ArgType
+import kotlinx.cli.default
+
+/**
+ * @param args CLI args
+ * @return an instance of [SaveConfig]
+ */
+fun createConfigFromArgs(args: Array<String>): SaveConfig {
```

Comment:
may be will add some argument for a baseline? (while I haven't yet forgotten about that)
We should have a baseline functionality

Reply:
Added this and addressed other comments, please take a look